player.py

```diff
@@ -0,0 +1,44 @@
+from maze_generation import *
+import pygame
+
+class Player(pygame.sprite.Sprite):
+    def __init__(self, image, size, padding, x_pos, y_pos):
+        super(Player, self).__init__()
+        self.surf = pygame.image.load(image).convert()
+        self.surf = pygame.transform.scale(self.surf, (size, size))
+        self.rect = self.surf.get_rect()
+        self.rect = self.rect.move(x_pos, y_pos)
+        self.step_len = size + padding
+        self.x_pos, self.y_pos = 0, 0
```

Comment:
Make names clear for understanding that self.x_pos != x_pos (in function arguments) & self.y_pos != y_pos (in function arguments)